lib/amazing_print/inspector.rb

```diff
@@ -151,6 +163,8 @@ def merge_options!(options = {})
     # predictable values
     #---------------------------------------------------------------------------
     def load_dotfile
+      return if @@dotfile # Load the dotfile only once.
```

Comment:
Is `@@dotfile` actually ever assigned to?

Reply:
Yeah in `#dotfile_readable?`
https://github.com/amazing-print/amazing_print/blob/f398c79d29c12168d6b353caab661195d0331f88/lib/amazing_print/inspector.rb#L158-L161